docs/index.rst

```diff
@@ -12,6 +7,114 @@ Contents:
    :maxdepth: 2
 
 
+Overview
+--------
+
+Format
+~~~~~~
+
+All response bodies are sent as JSON objects.
+
+.. TODO example success response once we have an API endpoint
+
+Where specified, query string parameters can be provided.
+
+.. TODO example request once we have an API endpoint that accepts parameters
+
+Request bodies for ``POST``, ``PUT``, ``PATCH`` and ``DELETE`` requests are expected to be JSON objects.
+
+.. TODO example request once we have an API endpoint
+
+Verbs
+~~~~~
+
+Unless otherwise specified, the following verbs are used for requests:
+
++--------+------------------------------------------------------------+
+| Verb   | Description                                                |
++========+============================================================+
+| GET    | Used for retrieving a resource                             |
++--------+------------------------------------------------------------+
+| POST   | Used for creating a resource                               |
++--------+------------------------------------------------------------+
+| PUT    | Used to replace a resouce                                  |
++--------+------------------------------------------------------------+
+| PATCH  | Used to partially update a resource with fields given in   |
+|        | the request body                                           |
++--------+------------------------------------------------------------+
+| DELETE | Used to delete a resource                                  |
++--------+------------------------------------------------------------+
```

Comment:
Seems a bit like overkill detailing this, but I'm hoping it will help us stick to using the right http verbs when we develop the API.